memmap.go

```diff
@@ -347,6 +347,24 @@ func (m *MemMapFs) Chtimes(name string, atime time.Time, mtime time.Time) error
 	return nil
 }
 
+func (m *MemMapFs) Chown(name string, uid, gid int) error {
+	name = normalizePath(name)
+
+	m.mu.RLock()
+	f, ok := m.getData()[name]
+	m.mu.RUnlock()
+	if !ok {
+		return &os.PathError{Op: "chown", Path: name, Err: ErrFileNotFound}
+	}
+
+	m.mu.Lock()
```

Comment:
If the SetUID and SetGID functions are using locks, we won't need this lock anymore.